django_outbox_pattern/decorators.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=protected-access
```

Comment:
question: What is the motivation to include this bypass? Not is possible to remove that?

Reply:
In this case I could move it to the lines where there is access, I cannot remove it because I need to access the _meta.